### PR TITLE
Update rdf lib

### PR DESCRIPTION
**Pull request by jschnasse**
Found some issue with free running dewey ids caused by multiple occurrences of dc:subject. In accordance to the context.json it is mandatory to have only one dc:subject that references a list of subjects. The dewey id must be part of that list. See hbz/lobid-resources#260

### CODE REVIEW

File: src/test/resources/reverseTest/output/nt/00223/TT002234459.nt [hunk 1]
```diff
@@ -1,94 +1,91 @@
...
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/medium> <http://purl.org/ontology/bibo/Map> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/medium> <http://rdvocab.info/termList/RDACarrierType/1018> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/subject> _:b13 .
 _:b13 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://d-nb.info/gnd/4049787-2> .
 _:b13 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b14 .
 _:b14 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://d-nb.info/gnd/4114067-9> .
 _:b14 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b15 .
 _:b15 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> _:b8 .
-_:b15 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b16 .
-_:b16 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://dewey.info/class/914/> .
```
Comment (dr0i) [inline src/test/resources/reverseTest/output/nt/00223/TT002234459.nt:18]: We saw this yesterdays as `<http://lobid.org/resources/TT002234459#!> <<http://purl.org/dc/terms/subject> <http://dewey.info/class/914/> .` didn't we @jschnasse ?

Comment (jschnasse) [reply src/test/resources/reverseTest/output/nt/00223/TT002234459.nt:18]: This is the result from the reverseTest, the free running dewey was left out by the new version of rdf4j. See my comment on lobid-resources https://github.com/hbz/lobid-resources/issues/260#issuecomment-281335745

Comment (jschnasse) [reply src/test/resources/reverseTest/output/nt/00223/TT002234459.nt:18]: You have to look up the original input files to understand what is going wrong [HT015082724](https://github.com/hbz/lobid-rdf-to-json/blob/master/src/test/resources/input/nt/01508/HT015082724.nt), [HT013304490](https://github.com/hbz/lobid-rdf-to-json/blob/master/src/test/resources/input/nt/01330/HT013304490.nt), [TT002234459](https://github.com/hbz/lobid-rdf-to-json/blob/master/src/test/resources/input/nt/00223/TT002234459.nt)

**Comment by dr0i** [comment]
With hbz/lobid-resources#260 the `subject` data is coherent again.

**Comment by dr0i** [comment]
If you want you can update the test data using the new master of `lobid-resources`.
+1

**Comment by jschnasse** [comment]
Hi @dr0i, I decided to not longer work on this repo. Instead I will depend directly to lobid-resources. Unfortunatly updating of the rdf lib in lobid-resources causes conflicts in transitive libraries.

**Comment by dr0i** [comment]
Closing this as we [switched to the official jsonld-library](https://github.com/hbz/lobid-resources/issues/940).

**Pull request closed without merging**